content/tutorials/use-simple-xrp-payments/reliable-transaction-submission.md

```diff
@@ -122,9 +126,13 @@ For each persisted transaction without validated result:
         If (server has continuous ledger history from the ledger when the
               transaction was submitted up to and including the ledger
               identified by LastLedgerSequence)
-            The transaction failed (2)
-            If appropriate for the application, submit with
-                new LastLedgerSequence and Fee
+
+            # Sanity check
+            If (sender account sequence > transaction sequence)
+                A different transaction with this sequence has a final outcome.
```

Comment:
Should this be _potentially_ different transaction? Perhaps the transaction executed and you didn't notice. I guess it depends on how you intend "_If (result appears in validated ledger)_" to be interpreted. Does it mean "in _a_ validated ledger" or "in the _newest_ validated ledger"?

Reply:
Ah, yes, "If (result appears in validated ledger)" should be "If (result appears in any validated ledger)"

Will update that. :+1: